stdlib/http/http.flux

```diff
@@ -1,5 +1,19 @@
 package http
 
+import "experimental"
+
 // Post submits an HTTP post request to the specified URL with headers and data.
 // The HTTP status code is returned.
 builtin post
+
+endpoint =  (url) =>
+    (mapFn) =>
+        (tables=<-) =>
+            tables
+                |> map(fn: (r) => {
+                    obj = mapFn(r)
```

Comment:
should be `mapFn(r: r)`.

Reply:
Technically its valid since its the shortcut syntax of calling a function with name args if the identifier is the name of the arg. But I agree, I'd like the stdlib to be more explicit. I'll change it.

Reply:
Oh I didn't realize functions supported the shorthand syntax. In that case no need to change.